src/fork.rs

```diff
@@ -271,18 +285,12 @@ impl ForkDetails {
 
     /// Returns transactions that are in the same L2 miniblock as replay_tx, but were executed before it.
     pub async fn get_earlier_transactions_in_same_block(&self, replay_tx: H256) -> Vec<L2Tx> {
-        let client = self.create_client();
-
-        let tx_details = client
-            .get_transaction_by_hash(replay_tx)
-            .await
-            .unwrap()
-            .unwrap();
+        let tx_details = self.fork_source.get_transaction_by_hash(replay_tx).unwrap();
```

Comment:
We should avoid using `.unwrap()`

Reply:
for now - there is no way to return an error in this method. I can change it in a separate PR (as I don't want to grow the context of this one).